### PR TITLE
Adding user risk + change password to v1

### PR DESCRIPTION
**Pull request by elcarlosadrian**
The team is ready to move user risk + change password to v1

### CODE REVIEW

File: api-reference/v1.0/resources/conditionalaccessconditionset.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ Represents the type of conditions that govern when the policy applies.
 |clientAppTypes|String collection| Client application types included in the policy. Possible values are: `all`, `browser`, `mobileAppsAndDesktopClients`, `exchangeActiveSync`, `easSupported`, `other`.|
 |locations|[conditionalAccessLocations](conditionalaccesslocations.md)| Locations included in and excluded from the policy. |
 |platforms|[conditionalAccessPlatforms](conditionalaccessplatforms.md)| Platforms included in and excluded from the policy. |
-|signInRiskLevels|String collection| Risk levels included in the policy. Possible values are: `low`, `medium`, `high`, `none`.|
+|signInRiskLevels|String collection| Sign-in risk levels included in the policy. Possible values are: `low`, `medium`, `high`, `none`.|
+|userRiskLevels|String collection| User risk levels included in the policy. Possible values are: `low`, `medium`, `high`, `none`.|
```
Comment (Lauragra) [inline api-reference/v1.0/resources/conditionalaccessconditionset.md:26]: You'll need an update to the changelog MD and JSON files to mention the introduction of the new property.

Comment (elcarlosadrian) [reply api-reference/v1.0/resources/conditionalaccessconditionset.md:26]: Done

File: concepts/changelog.md [hunk 2]
```diff
@@ -42,6 +42,13 @@ For a summary of the value of these API changes, as well as recent tools, compon
...
 |Addition|beta|Added [create](/graph/api/accessreviewscheduledefinition-create?view=graph-rest-beta&preserve-view=true), [delete](/graph/api/accessreviewscheduledefinition-delete?view=graph-rest-beta&preserve-view=true), [get](/graph/api/accessreviewscheduledefinition-get?view=graph-rest-beta&preserve-view=true), [list](/graph/api/accessreviewscheduledefinition-list?view=graph-rest-beta&preserve-view=true), [update](/graph/api/accessreviewscheduledefinition-update?view=graph-rest-beta&preserve-view=true) methods to the [accessReviewScheduleDefinition](/graph/api/resources/accessreviewscheduledefinition?view=graph-rest-beta&preserve-view=true) resource.|
 
+### Identity and access | Identity and sign-in
+
+| **Change type** | **Version** | **Description** |
+|:---|:---|:---|
+|Addition|v1.0|Added the **userRiskLevels** property to [conditionalAccessConditionSet](https://docs.microsoft.com/en-us/graph/api/resources/conditionalAccessConditionSet?view=graph-rest-1.0) resource|
```
Comment (Lauragra) [inline concepts/changelog.md:49]: Please add periods to the ends of the sentences and add the &preserve-view=true parameter to the links.

File: api-reference/v1.0/resources/conditionalaccessconditionset.md (hunk 1 above)
Comment (videor) [inline api-reference/v1.0/resources/conditionalaccessconditionset.md:26]: @elcarlosadrian  we don't have none as an option for User risk. We only have low, medium and high

File: api-reference/v1.0/resources/conditionalaccessgrantcontrols.md [hunk 3]
```diff
@@ -18,10 +18,19 @@ Represents grant controls that must be fulfilled to pass the policy.
...
 | termsOfUse | String collection | List of [terms of use](/graph/api/resources/agreement) IDs required by the policy. |
 
+### Special considerations when using `passwordChange` as a control
+
+Consider the following when you use the `passwordChange` control: 
+
+- `passwordChange` must be accompanied by `mfa` using an `AND` operator. This combination ensures that the password will be updated in a secure way.
+- `passwordChange` must be used in a policy containing `userRiskLevels`. This is designed to enable scenarios where users must use a secure change password to reset their user risk.
+- The policy should target `all` applications, and not exclude any applications.
+- The policy cannot contain any other condition.
```
Comment (videor) [inline api-reference/v1.0/resources/conditionalaccessgrantcontrols.md:32]: Should we say - The policy cannot contain any other condition except 'users', 'applications' and `userRiskLevels'. 

Within CA APIs we treat users and applications as part of conditions.

Comment (elcarlosadrian) [reply api-reference/v1.0/resources/conditionalaccessgrantcontrols.md:32]: ok

File: concepts/changelog.md (hunk 2 above)
Comment (elcarlosadrian) [reply concepts/changelog.md]: I reverted this file

**Review by Lauragra — APPROVED** [review]

**Pull request merged**